packages/marked-mathjax/src/index.ts

```diff
@@ -0,0 +1,77 @@
+import type { Marked } from 'marked';
+
+const startMath = /(\$|\\\(|\\\[)/;
+
+/**
+ * Adds an extension to marked to handle math in a manner compatible with our
+ * use of Mathjax in PrairieLearn.
+ *
+ * @param marked - The marked instance to extend.
+ * @param MathJax - The MathJax instance to use for rendering math. In
+ * client-side code, this is `window.MathJax` (ensure `Mathjax.typesetPromise()`
+ * has been resolved). In server-side code, this is the result of
+ * `require('mathjax-full').init()`.
```

Comment:
We're no longer using `mathjax-full`.

Reply:
Updated in 4d5ebe79d5c516b50025de71ee40b6829230c853.